### PR TITLE
Update Detector Parameter Descriptions

### PR DESCRIPTION
**Pull request by QuentinTorg**
This PR updates the comments about parameters in the apriltag detector to give the user a better understanding of valid ranges and default values

### CODE REVIEW

File: apriltag.h [hunk 1]
```diff
@@ -130,39 +130,56 @@ struct apriltag_detector
...
     float quad_sigma;
 
     // When non-zero, the edges of the each quad are adjusted to "snap
     // to" strong gradients nearby. This is useful when decimation is
     // employed, as it can increase the quality of the initial quad
     // estimate substantially. Generally recommended to be on (1).
+    // Very computationally inexpensive
     //
-    // Very computationally inexpensive. Option is ignored if
-    // quad_decimate = 1.
```
Comment (QuentinTorg) [inline apriltag.h:153]: Option is no longer ignored if quad decimate is 1: https://github.com/AprilRobotics/apriltag/blob/master/apriltag.c#L895

File: apriltag.h [hunk 2]
```diff
@@ -130,39 +130,57 @@ struct apriltag_detector
...
+    // in pixels.  Very noisy images benefit from non-zero values (e.g. 0.8).
+    // Gaussian blur kernel size is (int)(4*fabssigma)). Add 1 if even.
+    // Negaive values will sharpen the image instead of blur
+    // Sample kernel sizes are below:
+    //
+    // max sigma          kernel size
+    // 0.0 -> 0.499       1  (disabled)
+    // 0.5 -> 0.999       3
+    // 1.0 -> 1.499       5
+    // 1.5 -> 1.999       7
```
Comment (christian-rauch) [inline apriltag.h:158]: This table is a bit confusing. You are referring to the look-up table for equation "kernel_size = 4 * abs(quad_sigma)":
https://github.com/AprilRobotics/apriltag/blob/bee5efb1cc263ec1459bca46193e3428220d52ba/apriltag.c#L1020-L1027

But why this mapping from 0 to 0.499, 0.5 to 0.999, etc.? That looks like 0.0 gives you 0.499 and 0.5 gives you 0.999. Do you mean that all values in this range map to the same kernel size?

The notation is a bit confusing and I would personally prefer if this is just properly formalised. With an ASCII or LaTeX function.

**Review by christian-rauch — CHANGES_REQUESTED** [review]
Please merge your fixup commits. I.e. you have two commits that change the same part twice. Merge them to make the history more readable.

**Comment by christian-rauch** [comment]
@QuentinTorg Are you interested in continuing with this?

**Pull request closed without merging**